packages/vscode-extension/src/utils/try-to-catch.ts

```diff
@@ -0,0 +1,17 @@
+// COPIED FROM https://github.com/coderaiser/try-to-catch/blob/26a0ba95c71b7bdb710bb678732531dd31549c40/lib/try-to-catch.js due to import errors
+// See https://stackoverflow.com/q/68953594/13994294
+
+export default async function tryToCatch(fn: (...args: unknown[]) => any, ...args: any[]): Promise<[error: Error | null, result?: any]> {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/LinqLover/downstream-repository-mining/pullRequest?prid=8058375)
```suggestion
export default async function tryToCatch(fn: (...args: unknown[]) => unknown, ...args: any[]): Promise<[error: Error | null, result?: any]> {
```